pymongo/encryption_options.py

```diff
@@ -47,6 +47,7 @@ def __init__(
         kms_tls_options: Optional[Mapping[str, Any]] = None,
         crypt_shared_lib_path: Optional[str] = None,
         crypt_shared_lib_required: bool = False,
+        bypass_query_analysis: bool = False,
```

Comment:
This will conflict with Julius' auto encryption PR but it was required for the tests.